src/main/java/com/senlainc/warsaw/tyurin/util/propertyHandlers/PropertyReader.java

```diff
@@ -0,0 +1,48 @@
+package com.senlainc.warsaw.tyurin.util.propertyHandlers;
+
+import com.senlainc.warsaw.tyurin.util.Constants;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+public class PropertyReader {
+
+    private static PropertyReader INSTANCE;
+    private Properties properties;
+    private Map<String, String> propertiesMap;
+
+    private PropertyReader() {
+        properties = new Properties();
+        propertiesMap = new HashMap();
+        readProperties();
+    }
+
+    public static PropertyReader getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new PropertyReader();
+        }
+        return INSTANCE;
+    }
+
+    private void readProperties() {
+        try (InputStream input = PropertyReader.class.getClassLoader().getResourceAsStream(Constants.PATH_TO_PROPERTIES)) {
+            System.out.println("readProperties");
+            properties.load(input);
+            propertiesMap.putAll(properties.entrySet()
```

Comment:
Why create additional Object for properties? You already have Properties class here. Just use properties.getProperty(key).
No need for additional object